src/components/common/MediaCard.test.js

```diff
@@ -318,21 +318,5 @@ describe('<MediaCard />', () => {
         expect(wrapper.find(IconButton)).toHaveLength(0);
       });
     });
-
```

Comment:
Why did this disappear?

Reply:
Because Text is now a connected element, so in the snapshot we cannot `dive` anymore. That's why I deleted this test.